src/main/java/com/cs98/VerbatimBackend/controller/GroupChallengeController.java

```diff
@@ -0,0 +1,99 @@
+package com.cs98.VerbatimBackend.controller;
+
+import com.cs98.VerbatimBackend.misc.Status;
+import com.cs98.VerbatimBackend.model.*;
+import com.cs98.VerbatimBackend.repository.*;
+import com.cs98.VerbatimBackend.request.CustomChallengeRequest;
+import com.cs98.VerbatimBackend.request.StandardChallengeRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.ObjectUtils;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.sql.Date;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+public class GroupChallengeController {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UserGroupRepository userGroupRepository;
+
+    @Autowired
+    private GroupChallengeRepository groupChallengeRepository;
+
+    @Autowired
+    private CategoryRepository categoryRepository;
+
+    @Autowired
+    private QuestionRepository questionRepository;
+
+    @Autowired
+    private StandardChallengeRepository standardChallengeRepository;
+
+    @PostMapping("api/v1/createStandardChallenge")
+    public ResponseEntity<List<Question>> createStandardChallenge(@RequestBody StandardChallengeRequest request) {
+        User createdByUser = userRepository.findByUsername(request.getCreatedByUsername());
+        UserGroup group = userGroupRepository.findByName(request.getGroupName());
```

Comment:
Group names aren't necessarily unique. If we want to fetch a single specific group then we probably want to query on the group's id (or figure out some other way of uniquely identifying a group)